src/utils/config.ts

```diff
@@ -70,6 +70,7 @@ export const MAXIMUM_DRAIN_DELAY = Math.round(
 	(1 * MINUTES) / intVar('MINIMUM_DRAIN_RATE_PER_MINUTE', 500),
 );
 
+// must be max(1) if VPN_OPENVPN_PROTO =~ udp
 export const VPN_INSTANCE_COUNT = getInstanceCount('VPN_INSTANCE_COUNT');
```

Comment:
Should we actually add code for this?
```suggestion
export const VPN_INSTANCE_COUNT = getInstanceCount('VPN_INSTANCE_COUNT');
if (optionalVar('VPN_OPENVPN_PROTO') === 'udp' && VPN_INSTANCE_COUNT > 1) {
	throw new Error('VPN_INSTANCE_COUNT can't be > 1 when VPN_OPENVPN_PROTO is udp');
}
```
or
```suggestion
export let VPN_INSTANCE_COUNT = getInstanceCount('VPN_INSTANCE_COUNT');
if (optionalVar('VPN_OPENVPN_PROTO') === 'udp') {
	console.error('VPN_INSTANCE_COUNT can't be > 1 when VPN_OPENVPN_PROTO is udp! Falling back to 1.');
	VPN_INSTANCE_COUNT = 1;
}
```